watt.py

```diff
@@ -23,7 +23,7 @@
 parser = argparse.ArgumentParser()
 parser.add_argument("-w", "--workflow", help="Name of workflow(s) whose tests should be run.", nargs="+")
 parser.add_argument("-t", "--test", help="Specific name of test to run; only tests matching this name will be run.",
-                    nargs="*")
+                    nargs="+")
```

Comment:
This should be `*` I think, since 0 args for this flag should mean "run all tests for specified workflows".

Reply:
My thinking here is that the behavior of the `-w` and `-t` option should be the same.  IMO, `watt -t -e cromwell.jar` should be an error, not a run all tests.

Also, w/ `nargs="*"` `watt -t -e cromwell.jar` will currently run no tests, not all tests, since the `test` param will be `[]` not `None`.  With either `*` or `+`, `watt -w workflow -e cromwell.jar` will run all tests for the specified workflow.

Reply:
> I think it's more common to let -t be empty rather than -w empty

I agree in general, although ironically the github workflow yaml in this PR shows the exact opposite use case, lol.

I'm not sure I'm following the argument here though.  Wouldn't the right way to say "run all tests for a workflow" be to include the `-w` but not the `-t`?  ie `watt -w workflow -e cromwell.jar`, as opposed to `watt -t -w worklow -e cromwell.jar`?  

And currently (at least when I run it) `watt -t -w worklow -e cromwell.jar` runs no workflows, which isn't the behavior I think you are saying you want, unless I'm misunderstanding.

Reply:
Yes, I agree with your first syntax, i.e. `watt -w workflow -e cromwell.jar` should (morally) run all tests for `workflow`. If that's not the case now, I think it's worth patching to allow this behavior. I thought that's how it would work normally but might've made a mistake. I can change it (unless you want to throw that into one of the PRs as well)

Reply:
right now the only difference is that you can include `-t` followed by nothing, in which case no tests are run.  Using `*` instead of `+` just allows that "empty" behavior, which I think should be an error.  Both `*` and `+` will allow not including the flag at all, ie `watt -w workflow -e cromwell.jar`.  So changing `*` to `+` I think gives the behavior we want:
```
watt -e cromwell.jar : run all tests for all workflows
watt -w workflow -e cromwell.jar : run all tests for particular workflow
watt -t test -e cromwell.jar : run particular test for all workflows
watt -w workflow -t test -e cromwell.jar : run particular test for particular workflow
watt -w -e cromwell.jar : error
watt -t -e cromwell.jar : error
``` 

This change (changing from `*` to `+`) just gets the last command to be an error, where currently with `*` it will just run no tests but not indicate any syntax problem.

Reply:
Ah, I see I misunderstood. Yes, this looks good, thanks!